src/clib/lib/enkf/enkf_plot_tvector.cpp

```diff
@@ -89,11 +89,6 @@ double enkf_plot_tvector_iget_value(const enkf_plot_tvector_type *plot_tvector,
     return double_vector_iget(plot_tvector->data, index);
 }
 
-time_t enkf_plot_tvector_iget_time(const enkf_plot_tvector_type *plot_tvector,
-                                   int index) {
-    return time_t_vector_iget(plot_tvector->time, index);
```

Comment:
you can also remove plot_t_vector->time as it is not read anywhere else

Reply:
Nice catch, thanks.